### DISCUSSION TITLE
Can not delete task on board

### DISCUSSION THREAD

**Discussion by DanyRamdas** [stacks-task-manager/stacks2#17] [Bugs]
Deleting task can not be done and move or copy also grayed out. 
Even deleting from archive can not be done, nothing happens.

**Comment by DanyRamdas** [comment]
![image](https://user-images.githubusercontent.com/41296194/215332896-e9a8e155-00f6-48b0-a540-0fd1d4d37e6a.png)

**Reply by synapse** [reply]
Hey @DanyRamdas this should be fixed in the latest Beta

**Reply by DanyRamdas** [reply]
Yes it is tnx!
